package/kernel/mwlwifi/Makefile

```diff
@@ -15,9 +15,9 @@ PKG_LICENSE_FILES:=
 
 PKG_SOURCE_URL:=https://github.com/kaloz/mwlwifi
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_DATE:=2023-11-29
-PKG_SOURCE_VERSION:=ebf3167445f108346dcff9a31a708534c0bd7cc5
-PKG_MIRROR_HASH:=6dffc08208305366b9c25133b74693d5c2e544c1076b7b4ed846628c66be56c1
+PKG_SOURCE_DATE:=2024-04-19
```

Comment:
IIRC apk requires replacing - with .

Reply:
This is done while generating `PKG_VERSION` from `PKG_SOURCE_DATE`.

https://github.com/openwrt/openwrt/blob/ea609fe486a99af6540dcc7a18f0827481d0c4b0/include/download.mk#L13

I’m fine with replacing it directly, but is seems most packages use `YYYY-MM-DD` format for source date.

Reply:
Ah OK nvm.